.travis.yml

```diff
@@ -1,38 +0,0 @@
-language: python
-cache: pip
-
-script:
-  - tox
-
-install:
-  - pip install tox tox-travis
-
-matrix:
-  include:
-    - python: "3.6"
-    - python: "3.7"
-    - python: "3.8"
-    - python: "3.9"
-
-    - python: "pypy3"
-
-    # Documentation
-    - python: "3.9"
-      env: TOXENV=docs
-      before_install:
-        - sudo apt-get -y install libenchant1c2a
-    - python: "3.9"
-      env: TOXENV=linkcheck
-
-    # Linting
-    - python: "3.9"
-      env: TOXENV=examples
-    - python: "3.9"
-      env: TOXENV=lint
-
-services:
-  - mongodb
-
-notifications:
-  email: false
-  irc: "irc.freenode.org#factory_boy"
```

Comment:
I believe we're losing that part though :(

Reply:
That's true. Do you consider this a blocker?

There are some IRC actions available from GH marketplace:

https://github.com/marketplace?query=irc

However, they aren't very compelling to me. For one, they seem to require a lot of customization per message and situation while the Travis version automatically handled different scenarios. If you think one of these should be included, let me know and I can experiment with them.

I suppose the alternative would be to use an in house custom action or command line tool.